shared/devices/device-page/dumb.desktop.js

```diff
@@ -15,7 +15,8 @@ const map: DumbComponentMap<Render> = {
   mocks: {
     'Normal': {
       ...common,
-      isCurrent: true,
+      deviceID: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
```

Comment:
The revoke RPC takes a deviceID rather than a name, so adding that to the mocks here.